chia/wallet/nft_wallet/nft_wallet.py

```diff
@@ -794,14 +780,16 @@ async def generate_unsigned_spendbundle(
         new_owner: Optional[bytes] = None,
         new_did_inner_hash: Optional[bytes] = None,
         trade_prices_list: Optional[Program] = None,
+        nft_coin: NFTCoinInfo = None,
```

Comment:
Is there any reason for this parameter when you can just specify the coin in the `coins` argument?  It seems like all this does here is add the need for an extra `if`